src/services/kratos/index.ts

```diff
@@ -0,0 +1,194 @@
+import { getKratosConfig, getKratosMasterPhonePassword } from "@config"
+import {
+  AuthenticationKratosError,
+  IncompatibleSchemaUpgradeError,
+  KratosError,
+  LikelyNoUserWithThisPhoneExistError,
+  LikelyUserAlreadyExistError,
+  UnknownKratosError,
+} from "@domain/users/errors"
```

Comment:
I'm thinking these errors should probably be in their own `@domain/authentication` module not under `user`. The definition of the:
```
interface KratosService { .. }
```
If you create one should live there too.